src/main/java/com/epam/tc/hw/mobile/pageObjects/NativePageObject.java

```diff
@@ -4,61 +4,119 @@
-    WebElement registerPwd;
+    private WebElement androidRegPwd;
 
     @AndroidFindBy(id = "platkovsky.alexey.epamtestapp:id/registration_confirm_password")
-    WebElement registerConfirmPwd;
+    private WebElement androidRegConfirmPwd;
 
     @AndroidFindBy(id = "platkovsky.alexey.epamtestapp:id/register_new_account_button")
-    WebElement registerAccBtn;
+    private WebElement androidRegUserBtn;
 
     @AndroidFindBy(xpath = ".//android.view.ViewGroup/android.widget.TextView")
-    WebElement pageName;
+    private WebElement androidPageName;
+
+    @FindBy(xpath = ".//XCUIElementTypeSecureTextField")
+    private WebElement iosSignInPwd;
+
+    @FindBy(xpath = ".//XCUIElementTypeTextField")
+    private WebElement iosSignInEmail;
+
+    @FindBy(xpath = ".//*[@value= 'Sign In']")
+    private WebElement iosSignInBtn;
+
+    @FindBy(xpath = ".//*[@value= 'Register new account']")
+    private WebElement iosRegBtn;
+
+    @FindBy(xpath = ".//*[@value= 'user@example.com']")
+    private WebElement iosRegEmail;
+
+    @FindBy(xpath = ".//*[@value= 'TimApple']")
+    private WebElement iosRegName;
+
+    @FindBy(xpath = ".//*[@value= 'Required']")
+    private WebElement iosRegPwd;
+
+    @FindBy(xpath = ".//*[@value= 'Repeat please']")
+    private WebElement iosRegConfirmPwd;
+
+    @FindBy(xpath = ".//*[@type= 'XCUIElementTypeSwitch']")
+    private WebElement iosRegSwitch;
+
+    @FindBy(xpath = ".//*[@name= 'I read agreaments and agree wit it']")
+    private WebElement iosRegAgreament;
+
+    @FindBy(xpath = ".//*[@value= 'Register new account']")
+    private WebElement iosRegUserBtn;
+
+    @FindBy(xpath = ".//XCUIElementTypeNavigationBar/XCUIElementTypeStaticText")
+    private WebElement iosPageName;
 
     public NativePageObject(AppiumDriver<?> appiumDriver) {
         PageFactory.initElements( new AppiumFieldDecorator(appiumDriver), this);
     }
 
-    public void registerUser(String email, String name, String password) {
-        registerBtn.click();
-        registerEmail.sendKeys(email);
-        registerUsername.sendKeys(name);
-        registerPwd.sendKeys(password);
-        registerConfirmPwd.sendKeys(password);
-        registerAccBtn.click();
+    public void registerIosUser(String email, String name, String password) {
+        iosRegBtn.click();
+        iosRegPwd.sendKeys(password);
+        iosRegConfirmPwd.sendKeys(password);
+        iosRegEmail.sendKeys(email);
+        iosRegName.sendKeys(name);
+        iosRegSwitch.click();
+        iosRegAgreament.click();
+        iosRegUserBtn.click();
+    }
+
+    public void registerAndroidUser(String email, String name, String password) {
+        androidRegBtn.click();
+        androidRegEmail.sendKeys(email);
+        androidRegName.sendKeys(name);
+        androidRegPwd.sendKeys(password);
+        androidRegConfirmPwd.sendKeys(password);
+        androidRegUserBtn.click();
+    }
+
+    public void loginAndroidUser(String email, String password) {
```

Comment:
It would be good not to duplicate the code, instead try to unify it simply, for example using abstraction